crates/tuono/src/cli.rs

```diff
@@ -47,15 +47,58 @@ fn init_tuono_folder(mode: Mode) -> std::io::Result<()> {
     Ok(())
 }
 
-pub fn app() -> std::io::Result<()> {
+fn extract_port(addr: &str) -> &str {
+    addr.split(":").last().unwrap_or_else(|| {
+        eprintln!("Error: Failed to extract port from address {}", addr);
+        std::process::exit(1);
+    })
+}
+
+async fn check_ports(mode: Mode) -> std::io::Result<()> {
+    println!("\nChecking ports...");
```

Comment:
I'd avoid logging this. This might be something to log in the future when we will have a `--debug` flag for the CLI. This info doesn't give any value to the user IMHO